src/app/pages/MainArticles/components/MainArticles/MainArticles.component.spec.jsx

```diff
@@ -0,0 +1,63 @@
+import renderer from 'react-test-renderer';
+import React from 'react';
+import { compose } from 'redux';
+import { shallow, mount } from 'enzyme';
+import dayjs from 'dayjs';
+
+import {
+  withProviders,
+  withAdditionalProperties,
+  articleMock as article
+} from '@core/tests/theme-helpers';
+
+import MainArticles from './MainArticles.component';
+
+const fetchArticlesMock = jest.fn().mockReturnValue([
+  article
+]);
+
+const WrappedComponent = compose(
+  withProviders,
+  withAdditionalProperties({
+    fetchArticles: fetchArticlesMock,
+  }),
+)(MainArticles);
+
+describe('Articles component test suite', () => {
+  it('should match snapshot', () => {
+    const component = renderer.create(<WrappedComponent />);
+    const tree = component.toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should fetch articles', () => {
+    shallow(<WrappedComponent />);
+    expect(fetchArticlesMock.mock.calls.length).toBe(1);
+  });
+
+  it('should mount component with children', () => {
+    const text = 'Time for new information';
+    const wrapper = mount(<WrappedComponent />);
+    expect(wrapper.find('h1').text()).toEqual(text);
+  });
+
+  it('should render articles component', () => {
+    const wrapper = mount(<WrappedComponent />);
+    expect(wrapper.find('.main-page')).toBeTruthy();
+  });
+
+  it('should have these five nodes when mounting', () => {
+    const wrapper = mount(<WrappedComponent />);    
+    expect(wrapper.containsAllMatchingElements([fetchArticlesMock.mock])).toBeTruthy();
```

Comment:
im wondering what this really doing? Becasue fetchArticlesMock is just a array with nothing.

Reply:
the name is: 'should have these five nodes when mounting'

ideally will be to populate componnet with few items and check if they are rendered corrcetly.

Reply:
what do you mean by `five nodes`? since your mock is just a single element, what are these `nodes` exactly?